misk-clustering-dynamodb/src/main/kotlin/misk/clustering/dynamo/DynamoClusterConfig.kt

```diff
@@ -3,7 +3,8 @@ package misk.clustering.dynamo
 import wisp.config.Config
 
 data class DynamoClusterConfig @JvmOverloads constructor(
-  val table_name: String = "misk-cluster-members",
+  val appName: String = System.getenv("SERVICE_NAME") ?: "<invalid-service-name>",
```

Comment:
This feels a bit weird/brittle vs. injecting `@AppName` which would work with existing service/test patterns.